pkg/helpers/helpers.go

```diff
@@ -164,6 +164,11 @@ func ShellQuote(s string) string {
 	return `'` + strings.Replace(s, `'`, `'\''`, -1) + `'`
 }
 
+// WindowsEscape safely escapes strings with single quotes which need to be used as Powershell arguments
+func WindowsEscape(s string) string {
+	return strings.Replace(s, `'`, `''`, -1)
```

Comment:
I don't think Windows actually needs this escaped if I'm reading it correctly. However, arm templates can't handle a single quote. You could surround the whole string in double quotes and PowerShell will handle it ok.

Reply:
after discussion, I think this will be cleanest

`return "@\" + s + "\"@"`

Reply:
see https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_quoting_rules?view=powershell-6#here-strings